app/src/main/res/layout/activity_movie.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
give proper file name, this is movie details activity so it can be like "activity_movie_details.xml"